fetch-node.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var wrapFetchForNode = function (fetch) {
+    // Support schemaless URIs on the server for parity with the browser.
+    // https://github.com/matthew-andrews/isomorphic-fetch/pull/10
+    return function (url, options) {
+        if (/^\/\//.test(url)) {
```

Comment:
Sure, that change makes sense.

I don't feel strongly about whether or not this is included, but yeah, that package is pretty lean, so they must have added it for a reason.